mmv1/templates/terraform/examples/dns_managed_zone_quickstart.tf.erb

```diff
@@ -38,7 +38,7 @@ resource "google_compute_firewall" "default" {
 # to create a DNS zone
 resource "google_dns_managed_zone" "<%= ctx[:primary_resource_id] %>" {
   name          = "<%= ctx[:vars]['example_zone_googlecloudexample'] %>"
-  dns_name      = "googlecloudexample.com."
+  dns_name      = "qa.tf-test.club."
```

Comment:
It looks like this is within the ci test project - in general we try to avoid that for non-ephemeral resources, but I'm not exactly sure where this _should_ go. Would it be possible to fix the configuration for `googlecloudexample.com` instead of setting up a new one? Alternately, would it be possible to generate new ones randomly here instead of hardcoding?

go/terraform-team-owned

Reply:
It looks like the priority of the failed tests pumped to P1. I need to read through some dns documentation and seek for a solution.